src/Widgets/AppContainers/AbstractAppContainer.vala

```diff
@@ -375,7 +375,7 @@ namespace AppCenter {
         protected virtual void update_progress_status () {
             Idle.add (() => {
                 progress_bar.text = package.get_progress_description ();
-                cancel_button.sensitive = package.change_information.can_cancel;
+                cancel_button.sensitive = package.change_information.can_cancel && !package.changes_finished;
```

Comment:
Can you clarify this change here?

Reply:
This is a bug that I think exists in master where the cancel button in the update view is sensitive for packages that have finished updating, but other package updates are ongoing.

It's not really required as part of these changes, but just something I noticed while testing the `can_cancel` stuff.